src/rt/sched/schedulerthread.h

```diff
@@ -75,7 +75,11 @@ namespace verona::rt
     SchedulerStats stats;
 
     T* list = nullptr;
+
+    // The number of cowns in the per-thread list `list`.
     size_t total_cowns = 0;
+    // The number of cowns that have been collected in the per-thread list
```

Comment:
Why is this atomic if it is per thread?

Reply:
Other threads can collect it, but not remove from the list.  I will expand the comment